modules/nw-metalLB-monitor-upgrading.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="metalLB-operator-monitoring-upgrade-status_{context}"]
+= Monitoring upgrade status
+The best way to monitor MetalLB upgrade status is to watch the `ClusterServiceVersion` (CSV) `PHASE`.
+You can also monitor the CSV conditions in the web console or by running the `oc get csv` command.
+
+[NOTE]
+====
+The `PHASE` and conditions values are approximations that are based on available information.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Run the following command:
++
+[source,terminal]
+----
+$ oc get csv
+----
+
+. Review the output, checking the `PHASE` field. For example:
++
+[source,terminal]
+----
+VERSION     REPLACES                                         PHASE
+4.11.0      metallb-operator.4.10-nnnnnnnnnnnn               Installing
```

Comment:
We said the upgrade won't work, so I think the users will never see this combination

Reply:
This comment is still standing